src/opnsense/scripts/dns/unbound_dhcpd.py

```diff
@@ -129,8 +139,8 @@ def run_watcher(target_filename, domain):
                         syslog.LOG_NOTICE,
                         "dhcpd expired %s @ %s" % (cached_leases[address]['client-hostname'], address)
                     )
-                    unbound_control(['local_data_remove',  cached_leases[address]['client-hostname']])
                     fqdn = '%s.%s' % (cached_leases[address]['client-hostname'], domain)
+                    remove_rr += f'{fqdn}\n{ipaddress.ip_address(address).reverse_pointer}\n'
```

Comment:
shouldn't it drop the A record here too?

Reply:
It does there are two lines here, the reverse pointer for the PTR record and the FQDN for the A record.

Switching to a list will make this much clearer.